src/lib/expansion/expansion-panel-header.ts

```diff
@@ -101,6 +102,17 @@ export class MatExpansionPanelHeader implements OnDestroy {
     .subscribe(() => this._changeDetectorRef.markForCheck());
 
     _focusMonitor.monitor(_element.nativeElement, renderer, false);
+
+    // When a panel is hosted in an accordion, subscribe to the expand/collapse subject
+    if (this.panel.accordion) {
+      this._expandCollapseAllSubscription =
```

Comment:
Can you move the contents of this `if` block to a separate method? Something like `_subscribeToAccordionActions`